packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/createTransferConnectionFrom..st

```diff
@@ -0,0 +1,7 @@
+interaction
+createTransferConnectionFrom: startingNode
```

Comment:
Maybe this could be `createConnectionFrom:ofType:`, where the second argument is the class of the connection?